client/onboarding-prototype/types.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+
+export type OnboardingFields = {
+	email?: string;
+	'individual.first_name'?: string;
```

Comment:
I'm guessing that it wouldn't be possible to have `individual` as a key representing an object containing `first_name` and `last_name` here? I am assuming it's like this because of the dot notation to nested object logic, but just checking/confirming!

Reply:
That makes sense, thanks for explaining! 

I don't think it's too hard to read currently, and it sounds like there's a good reason to keep it as it is to avoid issues with nested state changes and enforcing types, so I'd vote for keeping it as it is. 🙂